src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -132,6 +132,7 @@ public function load(array $configs, ContainerBuilder $container)
                     'emacs' => 'emacs://open?url=file://%%f&line=%%l',
                     'sublime' => 'subl://open?url=file://%%f&line=%%l',
                     'phpstorm' => 'phpstorm://open?url=file://%%f&line=%%l',
+                    'phpstorm-mac' => 'phpstorm://open?file=%%f&line=%%l',
```

Comment:
Are we sure that this will only ever be supported on Mac only? If not, what about `phpstorm-official`, `phpstorm-core`, or something along those lines? (I'm not a phpstorm user, so not sure if that makes sense).